BankWallet/BankWallet/Core/Managers/AppVersionManager.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+class AppVersionManager {
+    private let systemInfoManager: ISystemInfoManager
+    private let localStorage: ILocalStorage
+
+    init(systemInfoManager: ISystemInfoManager, localStorage: ILocalStorage) {
+        self.systemInfoManager = systemInfoManager
+        self.localStorage = localStorage
+    }
+}
+
+extension AppVersionManager: IAppVersionManager {
+
+    func didFinishLaunching() {
```

Comment:
Please rename this method to more informative one. AppVersionManager does not need to know about app finished launching action I guess.